src/test/java/choorai/excuseme/lecture/ui/LectureControllerTest.java

```diff
@@ -47,14 +53,75 @@ void afterEach() {
         memberRepository.deleteAll();
     }
 
+    @Test
+    @DisplayName("전체 강의 목록을 조회한다.")
+    void find_all_lectures() throws Exception {
+        // given
+        final Member normalMember = Member.createNormalMember(
+            "a@email.com",
+            passwordEncoder.encode("password1@"),
+            "이름",
+            "MEN",
+            "19990101",
+            "01012345678"
+        );
+        memberRepository.save(normalMember);
+
+        IntStream.rangeClosed(1, 10).forEach(i -> {
+            lectureRepository.save(new Lecture("lecture" + i, "thumbnail" + i, "videoUrl" + i));
+        });
+
+        final String accessToken = jwtProvider.createToken("a@email.com", Role.USER);
+
+        // when
+        final ExtractableResponse<Response> response = RestAssured
+            .given().headers("Authorization", "Bearer " + accessToken)
+            .when().get("/lectures")
+            .then().statusCode(HttpStatus.OK.value())
+            .extract();
+
+        // then
+        final List<LectureResponse> lectureResponses = response.jsonPath().getList("", LectureResponse.class);
+        assertThat(lectureResponses.size()).isEqualTo(10);
+    }
+
+    @Test
+    @DisplayName("특정 강의 조회를 조회한다.")
+    void find_lecture() throws Exception {
+        // given
+        final Member normalMember = Member.createNormalMember(
+            "a@email.com",
+            passwordEncoder.encode("password1@"),
+            "이름",
+            "MEN",
+            "19990101",
+            "01012345678"
+        );
+        memberRepository.save(normalMember);
+
+        final Lecture newLecture = lectureRepository.save(new Lecture("lecture", "thumbnail", "videoUrl"));
+        final String accessToken = jwtProvider.createToken("a@email.com", Role.USER);
+
+        // when
+        final ExtractableResponse<Response> response = RestAssured
+            .given().headers("Authorization", "Bearer " + accessToken)
+            .when().get("/lectures/" + newLecture.getId())
+            .then().statusCode(HttpStatus.OK.value())
+            .extract();
```

Comment:
p2

```suggestion
        final LectureDetailResponse response = RestAssured
            .given().headers("Authorization", "Bearer " + accessToken)
            .when().get("/lectures/" + newLecture.getId())
            .then().statusCode(HttpStatus.OK.value())
            .extract().body().as(LectureDetailResponse.class);
```
이와 같이하면 한번에 controller에서 응답하는 dto를 반환하는 것이 더 깔끔하다고 생각하는데 어떻게 생각하시나요?

Reply:
오호 저는 extract() 메서드로 뽑고, jsonPath를 활용해서 뽑아보았는데, 이 방법이 훨씬 보기 좋군요!
한 번 반영해보겠습니다! :)